spec/services/semester_spec.rb

```diff
@@ -0,0 +1,78 @@
+describe Semester do
+  it "Checking autumn semester calculated current in semester's January" do
+    Timecop.freeze '2020-01-01' do
+      expect(Semester.new("201920201").to_s==Semester.current.to_s).to be true
```

Comment:
These expressions can be written as
`expect(Semester.new("201920201").to_s).to eq Semester.current.to_s`

or in case of we want to check if they're not equal

`expect(Semester.new("201920201").to_s).not_to eq Semester.current.to_s`
which I think is a bit cleaner and easier to read.